utils/whois/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=whois
+PKG_VERSION:=5.5.7
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/rfc1036/whois/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=ae389c1486cdeae99f5f02940f98f5d7ff1f08ac0f96810365159b27b0189b5e
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/whois
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Intelligent WHOIS client
+  DEPENDS:=
+  URL:=https://github.com/rfc1036/whois
+endef
+
+define Package/whois/description
+In 1999 I wrote this Whois client from scratch because the alternatives were
```

Comment:
Maybe one should remove the "I" here and rephrase it to passive:
"This Whois client was written from scratch in 1999 because the alternatives were obsolete or bloated."

(The reader will not understand to whom the "I" will correspond in our context here.)

Personally, though, I would just drop this first sentence as the following ones sufficiently explain what's necessary.

Reply:
done